### PR TITLE
Add xycenter and xysize, modify xsize and ysize

### PR DESCRIPTION
**Pull request by Gouvernathor**

### CODE REVIEW

**Comment by Gouvernathor** [comment]
Adresses #2895

File: renpy/config.py [hunk 1]
```diff
@@ -1177,6 +1177,9 @@ def say_attribute_transition_callback(*args):
 # Should Ren'Py raise exceptions when finding an image?
 raise_image_exceptions = True
 
+# Should the size transform property only accept numbers of pixels ?
+old_size_tpref = False
```
Comment (renpytom) [inline renpy/config.py:1181]: This should probably be positive (defaulting to true), and tpref is something that's only used in the documentation - it's short for transform property reference. How about:

```
relative_transform_size = True
```

**Review by renpytom — COMMENTED** [review]
Looks mostly good, though some names should be changed. 

This needs documentation and changelog entries, too.

**Comment by Gouvernathor** [comment]
I took the liberty to move `size` down a bit in the list of transform properties, to store it next to `maxsize`.
I'll mark this as ready once I have tested it.

**Comment by renpytom** [comment]
I merged this in, so that it gets built by the nightly build system, and so I can test it here. It looked ready to me.

**Pull request merged**